build.gradle.kts

```diff
@@ -82,6 +82,11 @@ subprojects {
         archiveClassifier.set(project.name)
         duplicatesStrategy = DuplicatesStrategy.FAIL
 
+        dependencies {
+            include(dependency("org.yaml:snakeyaml"))
+        }
+        relocate("org.yaml.snakeyaml", "net.minecrell.serverlistplus.core.lib.snakeyaml")
```

Comment:
BTW, can you put this next to prettytime below (inside the if statement)? Then the standalone SLP-Server can still shade it without relocation.

I would also appreciate if you remove the snakeyaml version from `Server/build.gradle.kts` so it's not older than what's in this file. I think Gradle should prefer the newer version but it's a bit confusing.

Reply:
Makes sense. I tested a bit with the Server module and noticed that it wasn't pulling in any of the Server dependencies. Its seems like as soon as you define even a single `include` in the `shadowJar` it will not include any of the `implementation` dependencies anymore. So the duplicate SnakeYAML version would probably still be needed :S

Reply:
> Its seems like as soon as you define even a single include in the shadowJar it will not include any of the implementation dependencies anymore.

Yes, you can either use shadowJar with an allow list (when you have any `include()` line) or a block list (when you only have `exclude()` lines). So the trick is to not define any `include()` lines for the Server, so that it includes everything except what's listed as `exclude()`.

It shouldn't be necessary to add an explicit `implementation` line for SnakeYAML in `Server/build.gradle.kts` since it should be also pulled in as transitive dependency of the core.

Reply:
> It shouldn't be necessary to add an explicit `implementation` line for SnakeYAML in `Server/build.gradle.kts` since it should be also pulled in as transitive dependency of the core.

You are right, didn't think of that.